pkg/networking/networking_test.go

```diff
@@ -316,7 +313,6 @@ func Test_AddUserAgent_MissingIntegrationEnvironment_FormattedCorrectly(t *testi
 		IntegrationVersion: integrationVersion,
 		OS:                 osName,
 		Arch:               arch,
-		ProcessName:        processName,
```

Comment:
This could be a problem, as this is public.

Reply:
So far it was only used in the CLI and I will adapt it there.